.claude/rules/hooks.md

```
@@ -7,17 +7,17 @@ paths:
 
 ## Purpose
 
-Hooks integrate with Claude Code's hook system to enable real-time agent communication (inbox checking, trajectory tracking).
+Hooks integrate with Claude Code's hook system for trajectory tracking and other lifecycle events.
 
 ## Directory Structure
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)